crawl-ref/source/view.cc

```diff
@@ -1519,8 +1519,14 @@ void view_add_glyph_overlay(const coord_def &gc, cglyph_t glyph)
     glyph_overlays.push_back({gc, glyph});
 }
 
+#ifdef USE_TILE
+#define CONS_UNUSED(x) x
+#else
+#define CONS_UNUSED(x)
+#endif
```

Comment:
Instead of making a new macro, I would changed the part of the function body that looks like
```
#ifdef USE_TILE
    tiles stuff
#endif
```
into
```
#ifdef USE_TILE
    tiles stuff
#else
    UNUSED(tile);
#endif
```
as this is what I've seen used elsewhere in the codebase.

Reply:
Fine with me, I just did the first thing that popped into mind as I'm not fully familiar with all the weirdnesses in this codebase. (I would actually have moved that macro into a header and added the other two variants, but specifically wanted comments like yours first.)